client/src/utils/socket-context.tsx

```diff
@@ -54,54 +45,59 @@ export const SocketGateway = ({ children }: PropsWithChildren<any>) => {
 	const [logs, setLogs] = useState<string[]>([]);
 	const [aprsPacket, aprsDispatch] = useReducer(aprsReducer, {} as IPacket);
 	const [loRaPacket, loraDispatch] = useReducer(loraReducer, {} as IPacket);
-	const socket = io('http://localhost:8086/toClient');
+	const [packetFrequency, setPacketFrequency] = useState<number>(3);
+	const [protocol, setProtocol] = useState<string>('APRS');
+
+	const port = 9193;
```

Comment:
Magic number would be good to be be in the `.env` file

Reply:
moved port to a .env .